tsconfig.json

```diff
@@ -7,6 +7,7 @@
       "dom"
     ],
     "types": [
+      "mocha",
```

Comment:
We should keep the order consistent between all the tsconfigs. It'll make scripting changes much easier.